activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb

```diff
@@ -115,6 +115,19 @@ def supports_explain?
         true
       end
 
+      def supports_explain_analyze?
```

Comment:
It seems like we're only using these for active record's tests? Is this the right place?

Reply:
Reworked this